shoop/notify/static_src/script-editor.js

```diff
@@ -78,7 +78,7 @@ function Controller() {
         }
     };
     ctrl.setStepItemEditorState = function(state) {
-        if(!!state) {
+        if(state) {
```

Comment:
Doesn't eslint care whether it is `if(state)` or `if (state)`? I see both styles being used and IMHO we should use only one (the `if (state)` one if you ask me).